.github/workflows/release-next-version.yml

```diff
@@ -1,21 +1,18 @@
 name: Release Next Version
```

Comment:
We should consider renaming release-next-version to something like update-scheduled-release-version